src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -384,5 +386,197 @@
                  Assert-MockCalled New-AzIntegrationAccountMap -Times 0
             }
         }
+        Context "Azure Integration Account Assemblies" {
+            It "Providing both assemblyFilePath and assembliesFolder should fail" {
+                # Arrange
+                $resourceGroupName = "rg-infrastructure"
+                $integrationAccountName = "unexisting-integration-account"
+                $assemblyFilePath = "$PSScriptRoot\Files\IntegrationAccount\Assemblies\LibraryThatDoesSomething.dll"
+                $assembliesFolder = "$PSScriptRoot\Files\IntegrationAccount\Assemblies\"
+
+                # Act
+                { 
+                   Set-AzIntegrationAccountAssemblies -ResourceGroupName $resourceGroupName -Name $integrationAccountName -AssemblyFilePath $assemblyFilePath -AssembliesFolder $assembliesFolder -ErrorAction Stop
```

Comment:
Why are we adding `-ErrorAction Stop` to this? Shouldn't we let it fail, regularly? Not sure if adding this would do anything else, since we're already in the Pester `| Should -Throw` stuff.

Reply:
Thats probably true, works the same when I remove the `-ErrorAction Stop`. 
Removed it in the latest commit 👍🏻